src/m365/aad/commands/user/user-set.ts

```diff
@@ -181,7 +308,20 @@ class AadUserSetCommand extends GraphCommand {
       'accountEnabled',
       'currentPassword',
       'newPassword',
-      'forceChangePasswordNextSignIn'
+      'forceChangePasswordNextSignIn',
+      'displayName',
+      'firstName',
+      'lastName',
+      'forceChangePasswordNextSignInWithMfa',
+      'usageLocation',
+      'officeLocation',
+      'jobTitle',
+      'companyName',
+      'department',
+      'preferredLanguage',
+      'managerUserId',
+      'managerUserName',
+      'removeManger'
     ];
```

Comment:
Instead of using this ever long growing list of options, let's use `this.addUnknownOptionsToPayload` to add unknown options to the request body.